Yoast/wordpress-seo
# Change primary button into upsell button in configuration wizard

## Context


* Changes the primary button in the configuration wizard into an upsell button

## Summary


This PR can be summarized in the following changelog entry:

* Changes the primary button in the configuration wizard into an upsell button
## Relevant technical choices:

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and the latest develop from the monorepo (make sure they are linked)
* Open the configuration wizard
* Step 7 below "Outrank the competition" should have a yellow upsell button
* The other button on this page should still be a grey secondary button.

## UI changes

## Documentation

## Quality assurance


Fixes https://yoast.atlassian.net/browse/FRO-116
